src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshot.cs

```diff
@@ -128,7 +126,7 @@ private DefaultProjectSnapshot(ProjectSnapshotUpdateContext update, DefaultProje
 
         public override bool IsUnloaded => HostProject == null && WorkspaceProject == null;
 
-        public override string LanguageVersion => HostProject?.LanguageVersion;
+        public override string LanguageVersion => HostProject?.Configuration.LanguageVersion.ToString();
```

Comment:
Might as well just nuke this. Don't want to have a ton of proxy properties until we see a need for them.

Reply:
Sure, that makes sense